### PR TITLE
Feature/create skeleton campaign

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

Create button on homepage menu that generates a skeleton campaign with landing, overview and resource pages and auto-populates images, page titles and headings.

### CODE REVIEW

File: contentPages/models.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from wagtail.snippets.edit_handlers import SnippetChooserPanel
 from wagtail.snippets.models import register_snippet
 from CMS.enums import enums
+from django.contrib.contenttypes.models import ContentType
```
Comment (mwnicks) [inline contentPages/models.py:13]: Looks like this import is added but never used

File: core/templates/create_campaign.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    {% trans "Creating Campaign" as editing_str %}
+    {% include "wagtailadmin/shared/header.html" with title=editing_str subtitle=image.title icon="form" %}
+
+    <form action="/admin/create-campaign/" method="POST">
+        {% csrf_token %}
+        <div class="row row-flush nice-padding">
+            <div class="col6">
+                <ul class="fields">
+                    <li>
+                        <label class="label no-float">Page title:</label>
```
Comment (mwnicks) [inline core/templates/create_campaign.html:33]: Thinking about it this label might make more sense as 'Campaign name'.

File: core/views.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+class CreateCampaignView(View):
+    template_name = 'create_campaign.html'
+
+    def get(self, request):
+        return render(request, self.template_name, status=200)
+
+    def post(self, request):
+        form = CreateForm(request.POST)
+        if form.is_valid():
```
Comment (mwnicks) [inline core/views.py:14]: There should be something to handle invalid forms.

File: contentPages/models.py (hunk 1 above)
Comment (NoelleDL) [reply contentPages/models.py:13]: Deleted!

File: core/templates/create_campaign.html (hunk 2 above)
Comment (NoelleDL) [reply core/templates/create_campaign.html:33]: Changed this.

**Pull request merged**